source/_docs/solr-drupal-8.md

```diff
@@ -16,44 +16,75 @@ This will provision Apache Solr containers for every environment for your site.
 
 ## Install Solr on Drupal 8
 
-### 1. Apply Upstream Updates
+### Apply Upstream Updates
 Use [one-click updates](/docs/upstream-updates) to make sure you are running the latest version of Drupal core.
 
-### 2. Install the Search API Pantheon Module Using Composer
+### Install the Search API Pantheon Module
 
 [Search API Pantheon](https://www.drupal.org/project/search_api_pantheon) is a contributed module supported by Pantheon that interfaces with Pantheon's Solr service. This module may be in beta, but you can use it.
 
-Composer is the best way to install this module because this module relies on [Solarium](http://www.solarium-project.org/). Solarium is a Solr client library for PHP and is not Drupal-specific. Run the following command from within the site's root directory to register Drupal.org as a provider of Composer packages:
+<!-- Nav tabs -->
+<ul class="nav nav-tabs" role="tablist">
+  <!-- Active tab -->
+  <li id="tab-1-id" role="presentation" class="active"><a href="#tab-1-anchor" aria-controls="tab-1-anchor" role="tab" data-toggle="tab">Manual Install</a></li>
 
-```
-composer config repositories.drupal composer https://packages.drupal.org/8
-```
+  <!-- 2nd Tab Nav -->
+  <li id="tab-2-id" role="presentation"><a href="#tab-2-anchor" aria-controls="tab-2-anchor" role="tab" data-toggle="tab">Composer</a></li>
 
-Next, require the Search API Pantheon module:
+</ul>
 
-```
-composer require "drupal/search_api_pantheon ~1.0" --prefer-dist
-```
+<!-- Tab panes -->
+<div class="tab-content">
+  <!-- Active pane content -->
+  <div role="tabpanel" class="tab-pane active" id="tab-1-anchor" markdown="1">
+
+1.  The Search API Pantheon Module requires the [Solarium](http://www.solarium-project.org/) PHP library. Refer to [Drupal's documentation](https://www.drupal.org/docs/8/modules/libraries-api-8x/installing-an-external-library-that-is-required-by-a-contributed) if you're unfamiliar with adding external libraries to Drupal.
```

Comment:
Suggest rephrasing this as a directive, and possibly swapping the order of steps so the module is installed first:
```
1. Install the [Search API Pantheon Module](https://www.drupal.org/project/search_api_pantheon){.external} from the Drupal admin interface or via Terminus. For instructions, see [Working in the WordPress Dashboard and Drupal Admin Interface](/docs/cms-admin).
2. Download and add the [Solarium](http://www.solarium-project.org/){.external} PHP library (a dependecy for the aforementioned module). 
```

Reply:
Whoops, I lost the link to the drupal.org doc on libraries in this comment by accident. Let's keep that.